types/user/userconfig.d.ts

```diff
@@ -11,13 +13,13 @@ declare module 'libsession_util_nodejs' {
       name: string;
       priority: number;
       url: string;
-      key: Uint8Array;
+      key: Uint8Array; // FixedSizeUint8Array<32>
```

Comment:
Can you not use your new types here?

Reply:
that type is actually not very convenient. I might need to tweak it. Like it's convenient to use while only using FixedSize arrays, but getting one from a Uint8array is annoying and getting the buffer from a FixedSizeUint9Array is annoying too

Reply:
I'll keep this opened until I change it.